docs/reference/expectation-suite-operations.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Expectation Suite Operations
+---
+
+
+:::info sam-notes
+
+I suggest renaming this to "Expectation Suites" or combining it with the Expectation Suites section under "Expectations"
+
+:::
+
```

Comment:
would like to address this too before merging

Reply:
GDOC-144 will handle this